src/main/java/com/jw/OrderApplication.java

```diff
@@ -1,13 +1,14 @@
 package com.jw;
 
+import io.quarkus.logging.Log;
 import io.quarkus.runtime.Quarkus;
 import io.quarkus.runtime.annotations.QuarkusMain;
 
 @QuarkusMain
 public class OrderApplication {
 
     public static void main(String... args) {
-        System.out.println("Running main method");
+        Log.info("Starting Quarkus.Main");
```

Comment:
FYI: You can use @Slf4j from lombok also, however, it's nice to know that there is this class in quarkus.